components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FtpSendDynamicAwareTest.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.camel.component.file.remote;
```

Comment:
This class looks like its not complete, or added in the git PR as a mistake

Reply:
Yes, the file was added by mistake. Removed it.